### PR TITLE
Changes required to allow for CORS

### PR DESCRIPTION
**Pull request by opiechow**
Also changes in the API itself :)

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -26,32 +26,58 @@ fn main() -> Result<(), Box<dyn Error>> {
...
 }
 
+fn options_responder() -> impl Responder {
+    return HttpResponse::Ok()
+        .header("Access-Control-Allow-Origin", "*")
+        .header("Access-Control-Allow-Methods", "POST, GET, OPTIONS")
+        .header("Access-Control-Allow-Headers", "X-PINGOTHER, Content-Type")
+        .header("Access-Control-Max-Age", "86400")
+        .status(http::StatusCode::from_u16(204).unwrap())
+        .finish();
```
Comment (bartOssh) [inline src/main.rs]: Nie musisz dawac return, pomin ; 
bardziej Rust way

**Comment by bartOssh** [comment]
Remove not needed shit and merge :+1:

**Comment by bartOssh** [comment]
Nice clean solution with CORS Mate! Like it

File: src/main.rs (hunk 1 above)
Comment (opiechow) [reply src/main.rs]: Welp, no return breaks the response :)

**Pull request merged**